backend/api/Areas/Property/Controllers/ParcelController.cs

```diff
@@ -62,6 +70,41 @@ public IActionResult GetParcel(int id)
             return new JsonResult(parcel);
         }
 
+        /// <summary>
```

Comment:
The associated Land forms require parcel detail data, but cannot search via id as they only have the PID/PIN. These APIs allow searching by pid/pin to get parcel detail data.

Reply:
I need more data then the search endpoints offer. I can either do this, or add more data to the search endpoints which seems bad, or make two calls to the api (one to search to get the id from the pid, and then another to get the parcel).